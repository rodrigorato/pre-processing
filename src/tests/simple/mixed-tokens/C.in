public class Foo {alias display = System.out.println;public static void vertical(String va, String new) {String y = new String("Hey, you! Get off of my cloud!");display(y);}}