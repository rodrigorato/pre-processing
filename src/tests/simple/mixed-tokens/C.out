public class Foo {public static void vertical(String va, String new) {String y = new String("Hey, you! Get off of my cloud!");System.out.println(y);}}